spec/elastic_apm/grape_spec.rb

```diff
@@ -3,26 +3,41 @@
 if defined?(Grape)
   RSpec.describe Grape do
     describe '.start' do
-      before do
+
+      before(:all) do
         class GrapeTestApp < ::Grape::API
           use ElasticAPM::Middleware
         end
-        ElasticAPM::Grape.start(GrapeTestApp, config)
       end
 
-      after do
-        ElasticAPM.stop
+      after(:all) do
         Object.send(:remove_const, :GrapeTestApp)
       end
 
       context 'with no overridden config settings' do
+        before do
+          ElasticAPM::Grape.start(GrapeTestApp, config)
```

Comment:
Would this not need to stub the request for CentralConfig? Or can we just _not care_ with these changes? (If so, 👍)

Reply:
On one hand, the more defensive code in `CentralConfig` makes it not entirely necessary to mock `CentralConfig` requests in test. On the other hand, it's kind of "messy" to not mock `CentralConfig`.

So I think I've talked myself into having both the defensive code in `CentralConfig` in case a test forgets to mock it and also mock `CentralConfig` in this grape spec for completeness. What do you think @mikker ?